model_info/enwiki.drafttopic.md

```diff
@@ -15,611 +15,947 @@ Model Information:
+		Culture.Media.Music                          0.993
+		Culture.Media.Radio                          0.999
+		Culture.Media.Software                       0.999
+		Culture.Media.Television                     0.996
+		Culture.Media.Video games                    0.999
+		Culture.Performing arts                      0.999
+		Culture.Philosophy and religion              0.994
+		Culture.Sports                               0.99
+		Culture.Visual arts.Architecture             0.996
+		Culture.Visual arts.Comics and Anime         0.999
+		Culture.Visual arts.Fashion                  1
+		Culture.Visual arts.Visual arts*             0.992
+		Geography.Geographical                       0.99
+		Geography.Regions.Africa.Africa*             0.998
+		Geography.Regions.Africa.Central Africa      1
+		Geography.Regions.Africa.Eastern Africa      1
+		Geography.Regions.Africa.Northern Africa     0.999
+		Geography.Regions.Africa.Southern Africa     1
+		Geography.Regions.Africa.Western Africa      1
+		Geography.Regions.Americas.Central America   0.998
+		Geography.Regions.Americas.North America     0.968
+		Geography.Regions.Americas.South America     0.997
+		Geography.Regions.Asia.Asia*                 0.985
+		Geography.Regions.Asia.Central Asia          1
+		Geography.Regions.Asia.East Asia             0.995
+		Geography.Regions.Asia.North Asia            1
+		Geography.Regions.Asia.South Asia            0.995
+		Geography.Regions.Asia.Southeast Asia        0.997
+		Geography.Regions.Asia.West Asia             0.995
+		Geography.Regions.Europe.Eastern Europe      0.994
+		Geography.Regions.Europe.Europe*             0.967
+		Geography.Regions.Europe.Northern Europe     0.982
+		Geography.Regions.Europe.Southern Europe     0.992
+		Geography.Regions.Europe.Western Europe      0.99
+		Geography.Regions.Oceania                    0.993
+		History and Society.Business and economics   0.993
+		History and Society.Education                0.996
+		History and Society.History                  0.991
+		History and Society.Military and warfare     0.994
+		History and Society.Politics and government  0.983
+		History and Society.Society                  0.99
+		History and Society.Transportation           0.996
+		STEM.Biology                                 0.99
+		STEM.Chemistry                               0.999
+		STEM.Computing                               0.999
+		STEM.Earth and environment                   0.998
+		STEM.Engineering                             0.998
+		STEM.Libraries & Information                 1
+		STEM.Mathematics                             1
+		STEM.Medicine & Health                       0.997
+		STEM.Physics                                 1
+		STEM.STEM*                                   0.984
+		STEM.Space                                   0.999
+		STEM.Technology                              0.997
 		-------------------------------------------  -----
-	f1 (micro=0.668, macro=0.643):
+	f1 (micro=None, macro=None):
```

Comment:
should this be `None`?

Reply:
Aha.  It's None because it can't be computed when any of the topics are None.  Regretfully, we don't have enough observations in Central Africa to get a score for it.